ddp-workspace/projects/ddp-rarex/src/assets/i18n/en.json

```diff
@@ -36,6 +33,281 @@
+      "List": [
+        {
+          "Title": "CHANGES TO THE SITE AND CONTENT; MODIFICATION OF THESE TERMS",
+          "Paragraphs": [
+            "RARE-X MAY MAKE CHANGES TO, AND/OR DISCONTINUE OR SUSPEND ACCESS TO THE SITES, CONTENT AT ANY TIME WITHOUT NOTICE. YOU AGREE THAT RARE-X WILL NOT BE LIABLE TO YOU OR TO ANY THIRD PARTY FOR ANY CHANGES, SUSPENSION OR DISCONTINUANCE OF THE SITES, AND/OR CONTENT (OR ANY PART THEREOF).",
+            "RARE-X RESERVES THE RIGHT TO CHANGE OR MODIFY THESE TERMS AT ANY TIME. IF RARE-X MAKES ANY SUCH CHANGES, RARE-X WILL POST UPDATED TERMS OF USE ON THIS SITE. IF WE MAKE ANY MATERIAL CHANGES TO THESE TERMS, WE WILL NOTIFY YOU BY POSTING A NOTICE OF THE CHANGES ON THE SITE. IT IS YOUR RESPONSIBILITY TO REGULARLY VISIT AND REVIEW THESE TERMS. IF ANY MODIFICATION IS UNACCEPTABLE TO YOU, YOUR SOLE REMEDY SHALL BE TO CEASE USING THE SITE. IF YOU DO NOT CEASE USING THE SITE(S) AND/OR CONTENT FOLLOWING THE DATE WE POST THE UPDATED TERMS, YOU WILL BE DEEMED TO HAVE ACCEPTED THE CHANGE. NOTE THAT, ANY MODIFICATIONS TO THESE TERMS WILL NOT ACT TO MODIFY EITHER: THE PRIVACY POLICIES REFERRED TO IN SECTION 8 BELOW; OR THE TERMS OF ANY CONSENT FORMS THAT YOU AGREE TO IF YOU DECIDE TO PARTICIPATE IN ANY RARE-X DATA COLLECTION PROGRAMS."
+          ]
+        },
+        {
+          "Title": "ACCESS AND USE OF SITE AND CONTENT",
+          "Paragraphs": [
+            "Subject to your compliance with these Terms, RARE-X grants you a limited, revocable, nonexclusive right to access and use the Sites and the Content (as defined below) solely for your own internal use.",
+            "RARE-X and its licensors retain all right, title and interest in and to the Sites, including, without limitation, any and all text, graphics, images, music, software, audio, video, works of authorship of any kind, and information or other materials that are posted, generated, provided or otherwise made available through the Site (collectively, the “<span class=\"bold underline\">Content</span>”). All RARE-X trademarks are strictly owned by RARE-X, and nothing in these Terms will be construed to transfer ownership rights or grant any permission, license or other rights to any RARE-X trademark without written authorization from RARE-X. The names of actual companies and products mentioned within the Sites and/or Content may be the trademarks of their respective owners. RARE-X reserves all rights and licenses not expressly granted to you in these Terms and no implied license is granted by RARE-X. The Sites its underlying technology, are protected by copyright, trademark, patent, intellectual property, and other laws of the United States and foreign countries. You agree not to remove, change or obscure any copyright, trademark, service mark or other proprietary rights notices incorporated in or accompanying the Sites and/or any Content."
```

Comment:
Doesn't look like valid JSON syntax... need to escape the double-quotes!

Reply:
Interesting, VS Code had no problems with this JSON file and [this](https://jsonlint.com/) service also found no issues.

Reply:
It might be because this sentence here is using the Unicode double-quote character rather than the ASCII one. If you have verified that the "bold underline" snippet in this sentence is working properly, then OK from me.